cl/people_db/templates/view_person.html

```diff
@@ -24,6 +24,68 @@
         });
     });
 </script>
+<script type="application/ld+json">
+      {
+        "@context": "http://schema.org",
+        "@type": "Person",
+        "honorificPrefix": "Judge",
+        "familyName": "{{ person.name_first }}",
+        "givenName": "{{ person.name_last }}",
+        "gender": "{% if person.gender == 'm' %}http://schema.org/Male{% else %}http://schema.org/Female{% endif %}",
+        "nationality": {
+          "@type": "Country",
+          "name": "United States of America"
+        },
+        {% if person.has_photo %}
+        "image":
+          {% with person.name_last|add:"-"|add:person.name_first|lower|slugify as img_path %}
+          {% if person.date_dob %}
+            "{% static "judge_pics/"|add:img_path %}-{% granular_date person 'date_dob' default="" iso=True %}.jpeg"
+          {% else %}
+            "{% static "judge_pics/"|add:img_path %}.jpeg"
+          {% endif %}
+        {% endwith %},
+        {% endif %}
+        {% if person.date_dob %}
+        "birthDate": "{% granular_date person 'date_dob' iso=True %}",
+        {% endif %}
+        {% if person.dob_city or person.dob_state %}
+        "birthPlace": {
+          "@type": "{% if person.dob_city %}City{% else %}State{% endif %}",
+          "name": "{% if person.dob_city %}{{ person.dob_city }}, {% endif %}{{ person.dob_state }}"
+        },
+        {% endif %}
+        "deathDate": "{% granular_date person 'date_dod' iso=True %}",
+        {% if person.dod_city or person.dod_state %}
+        "deathPlace": {
+          "@type": "{% if person.dod_city %}City{% else %}State{% endif %}",
+          "name": "{% if person.dod_city %}{{ person.dod_city }}, {% endif %}{{ person.dod_state }}"
+        },
+        {% endif %}
+        {% if political_affiliations.count > 0 %}
+        "affiliation": [{% for pa in political_affiliations %}
+          {
+            "@type":"Organization",
+            "name": "{{ pa.get_political_party_display }}"
+          }{%if not forloop.last%},{%endif%}
+        {% endfor %}],
+        {% endif %}
+        {% if educations.count > 0 %}
+        "alumniOf": [{% for e in educations %}
+          {
+            "@type": "OrganizationRole",
+            "alumniOf": {
+              "@type": "CollegeOrUniversity",
+              "name": "{{ e.school.name }}"
+            }
+            {% if e.degree_year %}
+            ,"startDate": "{{ e.degree_year }}"
```

Comment:
I think this should be `endDate`, shouldn't it?

Reply:
It's confusing. `startDate` indicates date when became an alumni or left the org/graduated. See example 2 [here](https://schema.org/CollegeOrUniversity).

Reply:
The examples I see there are for alumni startDate fields, not OrganizationRole startDate fields? Note the indentation?

Reply:
I'm not sure I understand your questions. Does this example clarify? This is example 2 from my link above. `startDate` is a property of the first `alumniOf`. The example on schema.org has incorrect indentation. I've fixed the indentation below.

> A JSON example of an OrganizationRole being used to qualify
> the 'alumniOf' property (which is inverseOf 'alumni').
> Note that we use startDate to indicate when the alumniOf
> situation began, which was the date of leaving the organization.

```json
{
  "@context": "http://schema.org",
  "@type": "Person",
  "name": "Delia Derbyshire",
  "sameAs": "http://en.wikipedia.org/wiki/Delia_Derbyshire",
  "alumniOf": {
    "@type": "OrganizationRole",
    "alumniOf": {
      "@type": "CollegeOrUniversity",
      "name": "University of Cambridge",
      "sameAs": "http://en.wikipedia.org/wiki/University_of_Cambridge"
    },
    "startDate": "1959"
  }
}
```

Reply:
Oooh, this is confusing as hell because there's an `alumniOf` field inside an `alumniOf` field. Great design microdata folks! Yes, that clarifies!